src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -93,13 +95,19 @@ const SignInWithFacebook = () => {
         email: response.email
       };
       
-      Auth.federatedSignIn('facebook', { token: accessToken, expires_at }, user)
-        .then(credentials => {
-          console.log(credentials);
-        });
+      getCognitoCredentials('facebook', { token: accessToken, expires_at }, user);
```

Comment:
nit: Since you wrapped `Auth.federatedSignIn()` with a function `federatedSignIn()`, I renamed it to `getCognitoCredentials()` as it could be confusing